packages/utils/src/internals/sitemap.ts

```diff
@@ -303,7 +310,8 @@ export async function* parseSitemap<T extends ParseSitemapOptions>(
                     }
 
                     if (error !== null) {
-                        if (error.type !== 'fetch' || options?.reportNetworkErrors !== false) {
+                        const shouldIgnoreError = error.type === 'fetch' && reportNetworkErrors;
```

Comment:
if the error originates in `fetch` and we want to `reportNetworkErrors`, then we `shouldntIgnoreThisError`, no? :sweat_smile:

Reply:
Yeah. The tests caught this too.